assets/src/scss/layout/navbar/_navigation-mobile-menu.scss

```diff
@@ -0,0 +1,223 @@
+#navigation-mobile-menu {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  top: 0;
+  inset-inline-start: -100%;
+  height: 100vh;
+  padding: $sp-2 $sp-3;
+  z-index: 9999;
+  overflow-y: auto;
+  background: var(--top-navigation--background, $dark-blue);
+
+  ul {
+    // Reset the default value
+    padding-left: 0;
```

Comment:
Do you need to set both `padding-left` and `padding-inline-start`?

Reply:
Yes, we need it because of `rtl` direction. By `default` the `ul` is already set the `padding-left` attributo to `2rem` by bootstrap.

Let see what is happening with the left space between the main wrapper and the list, if you don't override this value
<img width="666" alt="Screenshot 2022-03-03 at 15 50 55" src="https://user-images.githubusercontent.com/77975803/156633100-996e7be9-ce0b-47da-bf44-0edbe63d4355.png">
<img width="675" alt="Screenshot 2022-03-03 at 15 53 19" src="https://user-images.githubusercontent.com/77975803/156633110-aa4da156-909a-47a8-8d2f-57392a904b70.png">

Reply:
Hmmm yeah I see, I hadn't tested in RTL 🤔 I think it would be possible to just write `padding: 0;` though?